modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css

```
@@ -0,0 +1,37 @@
+.dashboard-sales-container {
```

Please, change the class's name.
```suggestion
.sales-dashboard-container {
```

---

Actually, we're using .`dashboard-sales-container` pattern.